src/building/new-target.md

```diff
@@ -94,6 +94,17 @@ adding `NEW_TARGET_OS` as `target_os`:
 + (Some(Mode::Std), "target_os", Some(&["watchos", "NEW_TARGET_OS"])),
 ```
 
+To use this target in bootstrap, we need to explicitly add the target triple to the `STAGE0_MISSING_TARGETS`
+list in `bootstrap/src/core/sanity.rs`. This is necessary because the default compiler bootstrap uses does not
```

Comment:
```suggestion
list in `src/bootstrap/src/core/sanity.rs`. This is necessary because the default compiler bootstrap uses does not
```
Just to make it clearer.

Reply:
Done https://github.com/rust-lang/rustc-dev-guide/compare/31d839376aaef45ff8a13722a4b7d128b640323a..a11ed34e997960b4a0ecdeba625ac1a8c5a36b08